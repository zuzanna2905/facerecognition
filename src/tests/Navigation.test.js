import {shallow} from 'enzyme';
import React from 'react';
import Navigation from '../component/Navigation/Navigation';

it('renders Navigation without crashing', () => {
    const initialProps = {
        onRouteChange: '', 
        isSignedIn: false
    }
    let  wrapper = shallow(<Navigation {...initialProps}/>);
    expect(wrapper).toMatchSnapshot();
});

it('route to signout', () => {    
    const initialProps2 = {
        onRouteChange: '', 
        isSignedIn: true
    }
    let wrapper2 = shallow(<Navigation {...initialProps2}/>);
    expect(wrapper2).toMatchSnapshot();
})