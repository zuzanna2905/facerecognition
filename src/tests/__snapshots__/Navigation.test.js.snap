// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Navigation without crashing 1`] = `
<nav
  style={
    Object {
      "display": "flex",
      "justifyContent": "flex-end",
    }
  }
>
  <p
    className="f3 link dim black underline pa3 pointer"
    onClick={[Function]}
  >
    Sing In
  </p>
  <p
    className="f3 link dim black underline pa3 pointer"
    onClick={[Function]}
  >
    Register
  </p>
</nav>
`;

exports[`route to signout 1`] = `
<nav
  style={
    Object {
      "display": "flex",
      "justifyContent": "flex-end",
    }
  }
>
  <p
    className="f3 link dim black underline pa3 pointer"
    onClick={[Function]}
  >
    Sing Out
  </p>
</nav>
`;
